website_sale_coupon_page/tests/test_website_sale_coupon_page.py

```diff
@@ -6,8 +6,10 @@
 from PIL import Image
 
 from odoo.tests.common import HttpCase
+from odoo.tests import tagged
 
 
+@tagged('post_install', '-at_install')
```

Comment:
Why it should be post-install?

Reply:
And what about this?

Reply:
This was included to avoid a warning during the tests.